src/app/mauro/sort.type.ts

```diff
@@ -18,15 +16,12 @@ limitations under the License.
 
 SPDX-License-Identifier: Apache-2.0
 */
-@Injectable({
-  providedIn: 'root',
-})
-export class SortService {
-  ascString(a: string, b: string): number {
+export class Sort {
+  static ascString(a: string, b: string): number {
     return a > b ? 1 : b > a ? -1 : 0;
   }
 
-  descString(a: string, b: string): number {
+  static descString(a: string, b: string): number {
```

Comment:
When I suggested this change, I actually meant making plain functions because I felt like a class was unecessary, like this:

```ts
export const compareAsc = (a: string, b: string) => {
  return a > b ? 1 : b > a ? -1 : 0;
}
```

But this is fine for now.

Reply:
If we add other sort methods, then this list will grow, so I think it makes sense to keep them together in a class.